save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt

```diff
@@ -72,6 +73,22 @@ val fossGraphCollectionView: FC<FossGraphCollectionViewProps> = FC { props ->
                         }
                     }
                 }
+                column(id = "language", header = "Language", { language }) { cellContext ->
+                    Fragment.create {
+                        td {
+                            +"${ cellContext.row.original.language }"
+                        }
+                    }
+                }
+                if (!publicTable) {
+                    column(id = "status", header = "Status", { status }) { cellContext ->
+                        Fragment.create {
+                            td {
+                                +"${cellContext.row.original.status}"
```

Comment:
To my mind status should not be upper case

Reply:
We now display the status in upper case everywhere. I'd leave it like that for the sake of variety.